src/data/resolvers/colony.ts

```diff
@@ -197,7 +225,7 @@ export const getProcessedDomain = async (
     name: domainName,
   } = subgraphDomain;
   let name: string | null = null;
-  let color: string | null = null;
+  let color: number | null = null;
```

Comment:
I think that it should still be string here because it will ben an enum in the library as per Raul's comment
https://github.com/JoinColony/ColonyEventMetadataParser/pull/1#discussion_r920357223